docs/reference/FacebookCanvasHelper.md

```diff
@@ -1,19 +1,16 @@
-<card>
 # Facebook\Helpers\FacebookCanvasHelper
 
 The `FacebookCanvasHelper` is used to obtain an access token or signed request when working within the context of an [app canvas](https://developers.facebook.com/docs/games/canvas).
```

Comment:
I think this link does not contain relevant content anymore. I'm not sure to what it should be changed though.